src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes/BaseNodeModel.tsx

```diff
@@ -4,8 +4,8 @@ import { DiagramEngine, NodeModel } from 'storm-react-diagrams'
 import { StandardIncomingPortModel, StandardOutgoingPortModel } from '../nodes/Ports'
 
 export class BaseNodeModel extends NodeModel {
-  public isStartNode = false
-  public isHighlighted = false
+  public isStartNode: boolean
+  public isHighlighted: boolean
```

Comment:
I prefer having those set to false instead of undefined, the type is not required since it's obvious that it's a boolean